src/pal/src/misc/cgroup.cpp

```diff
@@ -211,11 +345,15 @@ class CGroup
 
             if (strncmp(filesystemType, "cgroup", 6) == 0)
```

Comment:
Though the existing code doesn't support v2, it passes this condition.
Maybe we want to make this more strict?
`if (strcmp(filesystemType, "cgroup") || strcmp(filesystemType, "cgroup2"))`?
Since we're comparing with a literal using `strncmp` doesn't add much benefit over `strcmp`. Or maybe the compiler flags don't allow it...